src/main/java/org/frc5687/infiniterecharge/robot/subsystems/Spinner.java

```diff
@@ -1,62 +1,71 @@
 package org.frc5687.infiniterecharge.robot.subsystems;
 
-import com.revrobotics.CANSparkMax;
-import com.revrobotics.CANSparkMaxLowLevel;
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
 import com.revrobotics.ColorSensorV3;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
 import edu.wpi.first.wpilibj.I2C;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import org.frc5687.infiniterecharge.robot.Constants;
 import org.frc5687.infiniterecharge.robot.RobotMap;
 import org.frc5687.infiniterecharge.robot.commands.DriveSpinner;
-import org.frc5687.infiniterecharge.robot.subsystems.OutliersSubsystem;
 import org.frc5687.infiniterecharge.robot.util.OutliersContainer;
 
 public class Spinner extends OutliersSubsystem {
     private ColorSensorV3 _colorSensor;
-    private CANSparkMax _sparkMax;
+    private TalonSRX _motorController;
+    private DoubleSolenoid _solenoid;
 
-    public enum Color {
-        red(0),
-        green(1),
-        blue(2),
-        yellow(3),
-        unknown(4);
```

Comment:
FYI as part of this PR I removed this well intentioned but likely useless Color enum and cleaned up anything related.